File: TimeLessDrive/src/main/java/com/duy/carshowroomdemo/entity/Car.java
package com.duy.carshowroomdemo.entity;

import jakarta.persistence.*;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import java.util.List;

@Getter
@Setter
@Entity
@Table(name = "car")
public class Car {
    @Id
    @GeneratedValue(strategy = GenerationType.UUID)
    @Column(name = "id", nullable = false)
    private String id;

    @Column(name = "name", length = 100)
    private String name;

    @Column(name = "price")
    private Long price;

    @Column(name = "status", length = 50)
    private String status;

    @OneToOne(cascade = CascadeType.ALL, fetch = FetchType.LAZY, orphanRemoval = true)
    private CarDescription carDescription;

    @ManyToOne(cascade = CascadeType.PERSIST, fetch = FetchType.LAZY)
    private Showroom showroom;

    @OneToMany(mappedBy = "car", cascade = CascadeType.ALL, fetch = FetchType.LAZY, orphanRemoval = true)
    private List<CarImage> carImageList;

    @OneToMany(mappedBy = "car", fetch = FetchType.LAZY)
    private List<Invoice> invoiceList;

    @OneToOne(mappedBy = "car")
    private Post post;
}

File: OldCarShowroom/OldCarShowroom.Service/Models/Car.cs
using System;
using System.Collections.Generic;

namespace OldCarShowroom.Service.Models;

public partial class Car
{
    public string Id { get; set; } = null!;

    public string CarDescriptionId { get; set; } = null!;

    public string? ShowroomId { get; set; }

    public string? Name { get; set; }

    public long? Price { get; set; }

    public string? Status { get; set; }

    public virtual CarDescription CarDescription { get; set; } = null!;

    public virtual ICollection<CarImage> CarImages { get; set; } = new List<CarImage>();

    public virtual Invoice? Invoice { get; set; }

    public virtual ICollection<OffMeeting> OffMeetings { get; set; } = new List<OffMeeting>();

    public virtual Post? Post { get; set; }

    public virtual Showroom? Showroom { get; set; }
}
